src/Components/Aspire.RabbitMQ.Client/RabbitMQEventSourceLogForwarder.cs

```diff
@@ -50,7 +50,7 @@ private void LogEvent(EventWrittenEventArgs eventData)
                 (eventData.EventId == 1 && eventData.EventName == "Info") ||
                 (eventData.EventId == 2 && eventData.EventName == "Warn"));
 
-            _logger.Log(level, eventId, new EventSourceEvent(eventData), null, s_formatEvent);
+            _logger.Log(level, eventId, s_formatEvent(new EventSourceEvent(eventData), null));
```

Comment:
Please revert this change. This is now creating a closure (i.e. an allocation) on every log message.

Reply:
Instead, you want to change `EventSourceEvent` to return `0` from its Count and its indexer.